presto-main/src/main/java/com/facebook/presto/sql/relational/SqlToRowExpressionTranslator.java

```diff
@@ -912,9 +913,36 @@ protected RowExpression visitLikePredicate(LikePredicate node, Context context)
                 return likeFunctionCall(value, call(getSourceLocation(node), "LIKE_PATTERN", functionResolution.likePatternFunction(), LIKE_PATTERN, pattern, escape));
             }
 
+            RowExpression prefixOrSuffixMatch = generateLikePrefix(value, pattern);
```

Comment:
Can we just call it prefixMatch, as we don't do suffixes. And add a todo for suffixes?

Reply:
Done